app/views/carts/_cart_item.html.haml

```diff
@@ -0,0 +1,41 @@
+.container
+  .row
+    .title-bar
+      %h4 Cart
+- if @order_item.present? && @order_item.errors.any?
+  %ul
+    - @order_item.errors.full_messages.each do |msg|
+      %li= msg
```

Comment:
leave a new line here - use new lines more, it's helpful and makes code more readable :-)